rust-runtime/aws-smithy-types/additional-ci

```diff
@@ -12,4 +12,4 @@ echo "### Checking for duplicate dependency versions in the normal dependency gr
 cargo tree -d --edges normal --all-features
 
 echo "### Running api-linter"
-cargo +nightly-2022-03-03 api-linter --all-features
+cargo "+${RUST_NIGHTLY_VERSION}" api-linter --all-features
```

Comment:
Where does this var come from?

Reply:
It's an environment variable set in the build Docker image here: https://github.com/awslabs/smithy-rs/blob/main/tools/ci-build/base-image.dockerfile#L117

It just occurred to me that using it here will mean we can't directly invoke these additional-ci scripts anymore... What's your preference on having more locations to update vs. having to jump through hoops to run one of these test scripts? I'm leaning towards having more places to update until it's easier to run the Docker CI actions.

Reply:
I assumed we'd be using docker to run the actions locally. You can also define a default in this file with something like:
`RUST_NIGHTLY_VERSION=${blah:-$RUST_NIGHTLY_VERSION}`